src/redux/reducers/opponents.ts

```diff
@@ -1,15 +1,20 @@
 import { Reducer } from 'redux';
 import { OpponentActionType, OpponentAction } from '../actions/opponents';
 
-export interface OpponentState {
-  opponents: string[];
+export interface Opponent {
```

Comment:
Is this the right file to define an `Opponent` interface?j

Reply:
Probably not. I think it'll do for now though - anticipating a re-org later when we have a better overview of the project.